### ISSUE TITLE
Batchnorm fusion

### ISSUE THREAD

**Issue by janchk** [aifoundry-org/MHAQ#64]
In order to fully comply with fully-integer requirement, we need to liquidate all of the remaining pieces of FP values in  the network. One of them being running var and mean inside Batch Normalization module. The idea is to fuse BN module into a previous quantizable layer.
